src/components/slider/slider.stories.ts

```diff
@@ -7,7 +7,11 @@ import { html } from "../../../support/formatting";
 export default {
   title: "Components/Controls/Slider",
   parameters: {
-    notes: readme
+    notes: readme,
+    chromatic: {
+      // https://www.chromatic.com/docs/threshold
+      diffThreshold: Number(process.env.CHROMATIC_DIFF_THRESHOLD) || 0.3
```

Comment:
Do you think setting this globally would be too relaxed?

Reply:
I think 0.15 will probably be okay globally, its more than double the default value. I think we've been using the default this whole time because I didn't put the property in `chromatic: {}` 😅

I bumped the fallbacks in case the environment variable doesn't work, but I'm still attempting to tweak it from the GH Secret. And then I'll commit the value once we get something that works